scholarx/archive/styles.css

```
@@ -162,3 +163,30 @@ hr {
     max-width: 100%;
   }
 }
+
+.loader {
+  margin: auto;
+  border: 12px solid #EAF0F6;
+  border-radius: 50%;
+  border-top: 12px solid rgb(215, 215, 215);
+  width: 125px;
+  height: 125px;
```

Still to big @janithlahirukariyawasam 
```suggestion
  width: 75px;
  height: 75px;
```